manifests/p/Pallhed/LiveresultatDesktop/1.1.0/Pallhed.LiveresultatDesktop.locale.en-US.yaml

```diff
@@ -0,0 +1,9 @@
+PackageIdentifier: Pallhed.LiveresultatDesktop
+PackageVersion: 1.1.0
+PackageLocale: en-US
+Publisher: Pallhed
+PackageName: LiveresultatDesktop
+License: Freeware
+ShortDescription: Liveresultat Setup

```

Comment:
```
ShortDescription: Liveresultat Setup
```
Would you be able to write a better description for what this application is about?

Reply:
Yes, I'll add a better desc.